notation/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+import { install } from './install';
+import { sign } from './sign'
+import { verify } from './verify';
+
+async function run() {
+    try {
+        let command = taskLib.getInput('command', true);
+        switch (command) {
+            case 'install':
+                await checkAndInstall();
+                break;
+            case 'sign':
+                await checkAndInstall();
+                await sign();
+                break;
+            case 'verify':
+                await checkAndInstall();
```

Comment:
I'm a little bit hesitate about implicitly installing notation CLI during `sign` and `verify`, and looks like this behavior is not mentioned in the doc. IMO, installing notation CLI can only be done by the `install` command.

Reply:
We can have discussion with PMs.

Reply:
Updated the logic to remove `install` process from sign and verify commands.